### DISCUSSION TITLE
How do I add sounds to my project in bridge.?

### DISCUSSION THREAD

**Discussion by jason7141** [bridge-core/editor#578] [Q&A] [answered]
How does one go about adding new sounds to bridge?

**Comment by Joelant05** [comment, marked as answer]
To get the sound in your project, simply drag it from the file explorer into bridge and choose "Save to Project", or use the "Open File" action under "File" in the toolbar. Then you can register the sound to the game in the `sounds/sound_definitions.json` file. You can read more about adding sounds to Minecraft [here](https://wiki.bedrock.dev/concepts/sounds.html).

**Reply by jason7141** [reply]
This is what I get it wont 
![Screenshot_20220820-091024_Chrome](https://user-images.githubusercontent.com/105606159/185747724-09f8f51a-057b-4380-8e2c-5554ecf9020e.jpg)
create a sound file

**Reply by jason7141** [reply]
It won't create a sound file when I create the folder

**Reply by ihategravel2** [reply]
From your screenshot it seems you didn't save it to your project and instead just opened it. That gibberish your viewing when opening is the attempt to view the data as regular text.

**Reply by jason7141** [reply]
I found it thanks for your help! after you load it in you can click on file then open folder and open files you've downloaded then copy and paste in

**Comment by jason7141** [comment]
It must be a bug because it dosnt allow me to add the sounds into the file
